### DISCUSSION TITLE
Formula One Card isn't showing next race

### DISCUSSION THREAD

**Discussion by larsvdmanden** [marcokreeft87/formulaone-card#317] [Q&A]
Hi,

The Formula One Card does not show the first race that's coming. I selected a countdown to the qualifying, sprint and race. But this morning it was an hour too early for qualifying, and now he's showing already the next race in Abu Dhabi whilst we race tomorrow (for Europeans). When I try to select the types I want to see, the tickboxes don't look like they change anything and when I save a card and edit it again, all the boxes are empty again.

What's going wrong?
<img width="456" alt="Screenshot 2023-11-18 at 10 56 47" src="https://github.com/marcokreeft87/formulaone-card/assets/58793484/99dcb6aa-99ff-430c-b9c2-f45d19f12d54">
<img width="955" alt="Screenshot 2023-11-18 at 11 02 53" src="https://github.com/marcokreeft87/formulaone-card/assets/58793484/27827274-86c6-4fdf-b34c-ba8fb02a99ae">

The code:

      - type: custom:formulaone-card
        card_type: countdown
        f1_font: true
        tap_action:
          action: navigate
          navigation_path: /tablet-living-room/f1-stats
        show_raceinfo: false
        only_show_date: false
        show_weather: false
        card_mod:
          style: |
            ha-card {
            height: 100px;
            width: 236px;
            font-size: 6px;
            left: -85px;
            }

**Comment by marcokreeft87** [comment]
Yeah the API has the times all incorrect. Nothing I can do about this :(
Its going to be discontinued so they probably wont fix this
